### ISSUE TITLE
Use Rescuezilla Without Mouse 

### ISSUE THREAD

**Issue by GreenYun** [rescuezilla/rescuezilla#470]
I have used rescuezilla on my server, with only keyboard and monitor connected. Without a mouse, I cannot select Backup or Restore from the first UI of Rescuezilla with "Tab" key or arrow keys. "Tab" key could only move the highlight between "Clone" and "Verify Image" and "Back".

![image_2024-03-05_09-44-32](https://github.com/rescuezilla/rescuezilla/assets/26381267/1de49096-8aac-468a-ba2c-006d76c9c2fd)

**Comment by shasheene** [comment]
You're correct: Rescuezilla is not very usable when used with just a keyboard right now. I'll see what I can do to fix this.

Until this is improved with Rescuezilla, I recommend keyboard-only users stick with the existing Clonezilla app since it's highly-keyboard optimized (to the point of not supporting a mouse). Despite having other user-interface issues (no 'Back' button, confusing menus) it's a very reliable application.

It's worth noting for systems with only 1 USB port, booting with 'Load USB into RAM' will allow users to unplug the Rescuezilla USB stick and plug in a mouse.

Related issues:

https://github.com/rescuezilla/rescuezilla/issues/322 (another keyboard-only request)
https://github.com/rescuezilla/rescuezilla/issues/445 (screen-reader accessibility)

**Comment by GreenYun** [comment]
> You're correct: Rescuezilla is not very usable when used with just a keyboard right now. I'll see what I can do to fix this.
> 
> Until this is improved with Rescuezilla, I recommend keyboard-only users stick with the existing Clonezilla app since it's highly-keyboard optimized (to the point of not supporting a mouse). Despite having other user-interface issues (no 'Back' button, confusing menus) it's a very reliable application.
> 
> It's worth noting for systems with only 1 USB port, booting with 'Load USB into RAM' will allow users to unplug the Rescuezilla USB stick and plug in a mouse.
> 
> Related issues:
> 
> #322 (another keyboard-only request) #445 (screen-reader accessibility)

As per my test, the UX of the following steps is just OK for keyboard-only users, while the blue highlight is not so "high".

If UX improvements make development difficult, will adding a keyboard mouse functionality to the system be a good way?
